mods/cnc/maps/nod01/map.yaml

```diff
@@ -1,38 +1,35 @@
-MapFormat: 7
+Selectable: False
+
+MapFormat: 6
 
 RequiresMod: cnc
 
-Title: Nikoomba's Demise
+Title: Nikoomba's Demise (1)
 
 Description: In order for the Brotherhood to gain a foothold, we must begin by eliminating certain elements.\n\nNikoomba, the nearby village's leader, is one such element.\n\nHis views and ours do not coincide.\n\nHe and his whole group must be eliminated.
 
 Author: Westwood Studios
 
+PreviewVideo: nod1.vqa
+
 Tileset: DESERT
 
 MapSize: 64,64
 
 Bounds: 21,14,37,24
 
-Visibility: MissionSelector
+UseAsShellmap: False
 
 Type: Campaign
 
-Videos:
-	BackgroundInfo: intro2.vqa
-	Briefing: nod1.vqa
-	GameLost: nodlose.vqa
-
 Options:
 	Crates: False
-	Creeps: False
```

Comment:
Please don't delete this. It was introduced in https://github.com/OpenRA/OpenRA/pull/7220.